code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/filter/ssl/CertificateCreateValidationFilter.java

```diff
@@ -11,11 +11,11 @@
 import io.github.ibuildthecloud.gdapi.util.ResponseCodes;
 import io.github.ibuildthecloud.gdapi.validation.ValidationErrorCodes;
 
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
```

Comment:
YIKES!!.  We should a do a quick grep and see if commons logger is used anywhere else.